### PR TITLE
Replace Kapacitor Alert Endpoint Dropdown with Radio Buttons

### PR DESCRIPTION
**Pull request by lukevmorris**
- [x] CHANGELOG.md updated
  - [x] Rebased/mergable
  - [x] Tests pass
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

Connect #982 

### The problem
See #982

### The Solution
We replace the vertical Dropdown with a horizontal Radio Button Group:
screen shot 2017-04-11 at 4 36 51 pm

### CODE REVIEW

File: ui/src/kapacitor/components/RuleMessage.js [hunk 1]
```diff
@@ -55,17 +55,25 @@ export const RuleMessage = React.createClass({
...
         <div className="rule-section-body">
-          <div className="rule-section--item alert-message--endpoint">
-            <div>
-              <p>Send this Alert to:</p>
-              <Dropdown className="dropdown-kapacitor size-136" selected={selectedAlert || 'Choose an output'} items={alerts} onChoose={this.handleChooseAlert} />
-            </div>
+          <div className="kapacitor-values-tabs">
+            <p>Send this Alert to:</p>
+            <ul className="btn-group btn-group-lg tab-group">
+              {alerts.map(alert =>
```
Comment (cryptoquick) [inline ui/src/kapacitor/components/RuleMessage.js]: `text` could be destructured, as it's subsequently accessed three times.

File: ui/src/kapacitor/components/RuleMessage.js [hunk 2]
```diff
@@ -55,17 +55,25 @@ export const RuleMessage = React.createClass({
...
+                <li
+                  key={alert.text}
+                  className={classnames('btn tab', {active: alert.text === selectedAlert})}
```
Comment (cryptoquick) [inline ui/src/kapacitor/components/RuleMessage.js]: Good use of classnames.

File: ui/src/kapacitor/components/RuleMessage.js (hunk 1 above)
Comment (lukevmorris) [inline ui/src/kapacitor/components/RuleMessage.js]: If I need to access both `text` and the `alert` itself, how would that look?

Comment (cryptoquick) [inline ui/src/kapacitor/components/RuleMessage.js]: haha, you'd lose the implicit return by adding a block and a return and it's just not worth it at that point. good point!

**Pull request merged**